packages/flutter/lib/src/rendering/sliver_persistent_header.dart

```diff
@@ -394,15 +509,22 @@ class FloatingHeaderSnapConfiguration {
   /// Creates an object that specifies how a floating header is to be "snapped"
   /// (animated) into or out of view.
   FloatingHeaderSnapConfiguration({
-    @required this.vsync,
+    @Deprecated(
+      'Specify SliverPersistentHeaderDelegate.vsync instead. '
+      'This feature was deprecated after v1.19.0.'
+    )
+    this.vsync,  // ignore: deprecated_member_use_from_same_package
```

Comment:
I think we can remove this ignore, the analysis settings don't require it anymore.
```suggestion
    this.vsync,
```